pkg/asset/installconfig/aws/validation.go

```diff
@@ -317,6 +319,8 @@ func validateRegion(region string) error {
 func validateEndpointAccessibility(endpointURL string) error {
 	// For each provided service endpoint, verify we can resolve and connect with net.Dial.
 	// Ignore e2e.local from unit tests.
+	ctx, cancel := context.WithDeadline(context.TODO(), time.Now().Add(1*time.Minute))
```

Comment:
Can you explain the 1-minute timeout chosen? By default, the OS will impose a timeout on the dial (typically 3 minutes).

Reply:
I didn't know the OS did that. I'll change it to 3 minutes.

Reply:
My suggestion is that you omit the timeout entirely and leave it up to the OS defaults.